bookmarks/services/website_loader.py

```diff
@@ -34,11 +34,22 @@ def load_website_metadata(url: str):
 
 
 def load_page(url: str):
-    r = requests.get(url, timeout=10)
+    headers = fake_request_headers()
+    r = requests.get(url, timeout=10, headers=headers)
 
     # Use charset_normalizer to determine encoding that best matches the response content
     # Several sites seem to specify the response encoding incorrectly, so we ignore it and use custom logic instead
     # This is different from Response.text which does respect the encoding specified in the response first,
     # before trying to determine one
     results = from_bytes(r.content)
     return str(results.best())
+
+
+def fake_request_headers():
```

Comment:
For a future PR, why not make this a constant (since it's contents are always the same)?

Reply:
No reason. One article recommended to rotate the user agent, which would have required a function, but I never got that far. It could be changed to a constant.